openstack/compute/v2/extensions/remoteconsoles/requests.go

```diff
@@ -0,0 +1,83 @@
+package remoteconsoles
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// RemoteConsoleProtocol represents valid remote console protocol.
+// It can be used to create a remote console with one of the pre-defined protocol.
+type RemoteConsoleProtocol string
+
+const (
+	// RemoteConsoleVNCProtocol represents the VNC console protocol.
+	RemoteConsoleVNCProtocol RemoteConsoleProtocol = "vnc"
```

Comment:
How about `ConsoleProtocolVNC`. Same pattern for the rest.

Reply:
Seems better, yeah.